src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingViewExtensions.shared.cs

```diff
@@ -0,0 +1,200 @@
+
+			// add p2
+			smoothedPoints.Add(p2);
+		}
+
+		// add the last point
+		var last = currentPointsCopy[^1];
+		smoothedPoints.Add(last);
+
+		return smoothedPoints;
+	}
+
+	static Point GetIntermediatePoint(PointF p0, PointF p1, PointF p2, PointF p3, in float t, in float tt, in float ttt) => new()
+	{
+		X = 0.5f *
+			(2f * p1.X +
+				(p2.X - p0.X) * t +
+				(2f * p0.X - 5f * p1.X + 4f * p2.X - p3.X) * tt +
+				(3f * p1.X - p0.X - 3f * p2.X + p3.X) * ttt),
+		Y = 0.5f *
+			(2 * p1.Y +
+				(p2.Y - p0.Y) * t +
+				(2 * p0.Y - 5 * p1.Y + 4 * p2.Y - p3.Y) * tt +
+				(3 * p1.Y - p0.Y - 3 * p2.Y + p3.Y) * ttt)
+	};
+
+#if ANDROID || IOS || MACCATALYST || WINDOWS
+	/// <summary>
+	/// Set MultiLine mode
+	/// </summary>
+	/// <param name="mauiDrawingView"><see cref="MauiDrawingView"/></param>
+	/// <param name="multiLineMode">value</param>
+	public static void SetMultiLineMode(this MauiDrawingView mauiDrawingView, bool multiLineMode)
+	{
+		mauiDrawingView.MultiLineMode = multiLineMode;
+	}
+
+	/// <summary>
+	/// Set DrawAction action
+	/// </summary>
+	/// <param name="mauiDrawingView"><see cref="MauiDrawingView"/></param>
+	/// <param name="draw">value</param>
+	public static void SetDrawAction(this MauiDrawingView mauiDrawingView, Action<ICanvas, RectF>? draw)
+	{
+		mauiDrawingView.DrawAction = draw;
+	}
+
+	/// <summary>
+	/// Set ClearOnFinish
+	/// </summary>
+	/// <param name="mauiDrawingView"><see cref="MauiDrawingView"/></param>
+	/// <param name="clearOnFinish">value</param>
+	public static void SetClearOnFinish(this MauiDrawingView mauiDrawingView, bool clearOnFinish)
+	{
+		mauiDrawingView.ClearOnFinish = clearOnFinish;
+	}
+
+	/// <summary>
+	/// Set LineColor
+	/// </summary>
+	/// <param name="mauiDrawingView"><see cref="MauiDrawingView"/></param>
+	/// <param name="lineColor">line color</param>
+	public static void SetLineColor(this MauiDrawingView mauiDrawingView, Color lineColor)
+	{
+		mauiDrawingView.LineColor = lineColor;
+	}
+
+	/// <summary>
+	/// Set LineWidth
+	/// </summary>
+	/// <param name="mauiDrawingView"><see cref="MauiDrawingView"/></param>
+	/// <param name="lineWidth">line width</param>
+	public static void SetLineWidth(this MauiDrawingView mauiDrawingView, float lineWidth)
+	{
+		mauiDrawingView.LineWidth = lineWidth;
+	}
+
+	/// <summary>
+	/// Set Paint
+	/// </summary>
+	/// <param name="mauiDrawingView"><see cref="MauiDrawingView"/></param>
+	/// <param name="background">background</param>
+	public static void SetPaint(this MauiDrawingView mauiDrawingView, Paint background)
+	{
+		mauiDrawingView.Paint = background;
+	}
+
+	/// <summary>
+	/// Set Lines
+	/// </summary>
+	/// <param name="mauiDrawingView"><see cref="MauiDrawingView"/><see cref="MauiDrawingView"/></param>
+	/// <param name="drawingView"><see cref="IDrawingView"/></param>
+	public static void SetLines(this MauiDrawingView mauiDrawingView, IDrawingView drawingView)
+	{
+		if (mauiDrawingView.Lines.Count == drawingView.Lines.Count)
```

Comment:
we should compare 2 collections
We may have the same number of lines but it will be 2 different lines